tests/test_admin.py

```diff
@@ -190,46 +190,45 @@ def test_validation_filters(admin_client: Client) -> None:
         receipt=failed_validation_receipt,
     )
 
+    html = (
+        '<input class="action-select" name="_selected_action" value="{}" '
+        'type="checkbox" aria-label="Select this object for an action - {}">'
```

Comment:
I think that this should work on Django 5.1 and older versions too:

```suggestion
        'type="checkbox"
```

Reply:
it was failing for me until i checked the response and added that missing attribute :sweat_smile:
maybe there's something wrong with my environment/setup :(

Reply:
thanks! i tried different solutions and `test_validation_filters` now passes with the changes pushed in commit e1504564f467ce4776300b903dee12436e01052b for the different django versions. now, it seems that `test_html_view` is failing the same way as commit 7272160d7b8122d21f70064b29f56087d429f59e (https://builds.sr.ht/~whynothugo/job/1212081#task-tox-mariadb-79). i'll see if i can figure out a fix and push it as soon as i can :)